src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ObjectWriter.cs

```diff
@@ -282,12 +282,12 @@ public void BuildCFIMap(ObjectNode node)
             _offsetToCfiEnd.Clear();
             _frameOpened = false;
 
-            if (!(node is INodeWithFrameInfo))
+            if (!(node is INodeWithCodeInfo))
```

Comment:
Nit: You could do this to avoid the following cast:
```
INodeWithCodeInfo nodeWithCodeInfo = node as INodeWithCodeInfo;
if (nodeWithCodeInfo == null)
```